package com.riwi.codewordle.domain.entities;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class Word {
    private Long id;
    private String word;
    private Long themeId;
    private String hint;
    private LocalDateTime createdAt;
}
